consumer/src/main/kotlin/streams/StreamsEventSinkQueryExecution.kt

```diff
@@ -12,6 +12,7 @@ class StreamsEventSinkQueryExecution(private val streamsTopicService: StreamsTop
         if (cypherQuery == null) {
             return
         }
+        log.info("UNWIND ${params.size} events from Kafka")
```

Comment:
shouldn't this be a debug output?
with if (log.isDebugEnabled) around it.

Reply:
change the message to "Processing ...."

Reply:
Updated PR